conf.py

```diff
@@ -81,5 +67,49 @@
     "https://icsi.berkeley.edu*",  # Because it's broken often
 ]
 
+
+# -- Custom scripts ---------------------------------------------------
+
+# Hacky solution to list the latest engineering team members as reference
+# This grabs the latest list of team members from https://2i2c.org/about/
+# It then parses the HTML to grab the names and GitHub handles, then sorts by name
+# We can use this to define the order of team roles.
+from urllib import request
+from bs4 import BeautifulSoup
+import pandas as pd
+from pathlib import Path
+
+resp = request.urlopen("https://2i2c.org/about/")
+text = resp.read().decode()
+
+teams = []
+for row in BeautifulSoup(text, features="html.parser").select("div.people-widget > div"):  
```

Comment:
yeah I agree - I figured since this information was already embedded in our website I could try to re-use it.

Another option would be to create a Google Sheet like `2i2c Directory`. And it would list each person, their affiliation, their github / email handle, and any 2i2c groups they belong to. This would then be the canonical list of "who is in 2i2c?".

It would be another thing to maintain, but we probably need something like this.

Reply:
> It would be another thing to maintain, but we probably need something like this.

Yep, I would prefer just one sheet and then asking for that sheet info from whatever place we need it.
But again, we can do this later, it is not a blocker for this PR, IMHO.